api-schema/src/main/java/org/stellar/anchor/api/platform/PlatformTransactionData.java

```diff
@@ -30,6 +30,8 @@ public class PlatformTransactionData {
   Kind kind;
   SepTransactionStatus status;
 
+  String type;
+
```

Comment:
Whats the reason for adding a type field? I thought we were moving away from SEP-12 customer types?

Reply:
This is the `withdraw` and `deposit` type, like `bank_account` or `SWIFT`.